assets/templates/aws.vpcflow/vpcflow.conf.yml

```diff
@@ -0,0 +1,25 @@
+- name: Version
```

Comment:
This file does not have to be specified when running the tool. Is it picked up by convention? Why is it separate from the fields.yml? In case different configs should be specified?

Reply:
This file has to be manually passed to the generator via the `--config-file` flag https://github.com/elastic/elastic-integration-corpus-generator-tool/blob/main/cmd/generate.go#L91

Support for this is introduced in #36 (on top of which this PR is based).